src/nvim/change.c

```diff
@@ -224,7 +224,7 @@ static void changed_common(linenr_T lnum, colnr_T col, linenr_T lnume, long xtra
       // values for the cursor.
       // Update the folds for this window.  Can't postpone this, because
       // a following operator might work on the whole fold: ">>dd".
-      linenr_T last = lnume + xtra - 1;  // last line after the change
+      linenr_T last = lnume + (linenr_T)xtra - 1;  // last line after the change
```

Comment:
Can this `xtra` be changed to `linenr_T`?

Reply:
Nice catch.